### PR TITLE
added more graphs on moreinfo page

### PR DESCRIPTION
**Pull request by Zolisa001**
@miltontony : Created a pull request, not ready for review yet.

### CODE REVIEW

File: fridgedoor/static/js/moreinfo.js [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+for (i=0; i < profiles.length ; i++){
+
+	if ((profiles[i]).id === profile.id){
+		countryName = profiles[i].label;
+		break;		
+	}
+}
+$('#heading').html(countryName);
+
+console.log(heading);
```
Comment (miltontony) [inline fridgedoor/static/js/moreinfo.js]: some more cleanup here

File: fridgedoor/static/js/pageviews_monthly.js [hunk 2]
```diff
@@ -16,12 +16,141 @@ gapi.analytics.ready(function() {
...
+
+      var data1 = results[0].rows.map(function(row) { return +row[2]; });
+      var data2 = results[1].rows.map(function(row) { return +row[2]; });
+      var labels = results[1].rows.map(function(row) { return +row[0]; });
+
+      labels = labels.map(function(label) {
+        return moment(label, 'YYYYMMDD').format('ddd');
+      });
+
+      console.log("I have arrived");
```
Comment (miltontony) [inline fridgedoor/static/js/pageviews_monthly.js]: cleanup here too

File: fridgedoor/templates/pageviews_monthly.jinja2 [hunk 3]
```diff
@@ -20,4 +21,5 @@
 
 {% block js %}
   <script src="/static/js/pageviews_monthly.js?v=1"></script>
-{% endblock %}
+  <script src="/static/js/pageviews_weekly.js?v=1"></script>
```
Comment (miltontony) [inline fridgedoor/templates/pageviews_monthly.jinja2]: not sure why you added this here. Please remove

File: fridgedoor/templates/pageviews_weekly.jinja2 [hunk 4]
```diff
@@ -20,4 +20,5 @@
 
 {% block js %}
   <script src="/static/js/pageviews_weekly.js?v=1"></script>
-{% endblock %}
+  <script src="/static/js/pageviews_monthly.js?v=1"></script>
```
Comment (miltontony) [inline fridgedoor/templates/pageviews_weekly.jinja2]: same here. Please remove

Comment (Zolisa001) [inline fridgedoor/templates/pageviews_weekly.jinja2]: Okay will do just so.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**